Tails specific: we use kexec on both halt (runlevel 0) on top of reboot
(runlevel 6).

--- chroot.orig/etc/init.d/kexec-load  2011-01-14 12:30:05.089859516 +0100
+++ chroot/etc/init.d/kexec-load       2011-01-14 12:30:29.159667183 +0100
@@ -8,1 +8,1 @@
-# Default-Stop:		6
+# Default-Stop:		0 6
